devops/README.md

```diff
@@ -1,17 +1,147 @@
+# Package Installation
+We use the [Microsoft repository](https://packages.microsoft.com/) to distribute our packages. Below are the instructions to install the `osconfig` package on various operating systems.
 
-This folder contains the artifacts and scripts used by our engineering systems (build, test, package).
+## Amazon Linux 9 - insiders-fast (AMD64)
+```bash
```

Comment:
Do a table instead of sections?

Table with 3 columns:

Distro | Installation | Uninstallation